summary_workflows/differential_usage/D2_benchmark/D2_benchmark_specification.md

```diff
@@ -0,0 +1,92 @@
+# D2 benchmark specification
+
+## Synopsis
+
+Benchmark to test sensitivity and False Discovery Rate (FDR) of RNAseq-based identification of genes with differentially used poly(A) sites compared with ground truth (genes identified from orthogonal 3'end seq data).
+
+Input data:
+
+1. List of genes with information on differentially used poly(A) sites identified from RNA-Seq data using the benchmarked tool
+2. List of genes with information on differentially used poly(A) sites identified from orthogonal 3'end seq dataset
+
+Based on the input data the following metrics are computed:
+
+1. Sensitivity = (TP/(TP+FN))
+2. FDR = (FP/(TP+FP))
+
+TP - true positives - genes identified by the tool and present in the orthogonal dataset  
+FP - false positives - genes identified by the tool and not present in the orthogonal dataset  
+FN - false negatives - genes not identified by the tool but present in the orthogonal dataset
+
+
+## General info
+
+* **Challenge:** Differential usage
+* **Identifier:** D2
+
+## Inputs
+
+| # | Format | Link | Example data |
+  | --- | --- | --- | --- |
+  | 1 | CSV | [Specification][spec-csv] | [Link][in1] |
+  | 2 | CSV | [Specification][spec-csv] | [Link][in2] |
+
+### Additional info inputs
+  
+#### Format 1
+
+This CSV file contains a list of genes with differentially used poly(A) sites identified from RNA-Seq data by the benchmarked method.  
+Columns:
+
+- gene name
+- is_de: information whether **any** PAS in the gene was differentially expressed; possible values: "0" if no differential expression detected, "1" if differentially expressed
+- is_lengthened: information whether shortening/lengthening events were detected for the gene; possible values: "0" if no shortening/lengthening events detected, "1" if shortening/lengthening events detected
```

Comment:
I agree with the terminology. I would however, include the quantitative measure of usage and not just the p-value. This would mean then that we will have to define and calculate the measure of shortening/lengthening for a gene. But given that a gene can have multiple terminal exons and different tools could call different sites in different terminal exons, it makes more sense to do the analysis at the exon level.

Reply:
In terms of shortening/lengthening definition, I would probably take the one from LABRAT.